themes/custom/ts_wrin/js/components/ts_header_nav.js

```diff
@@ -268,11 +292,11 @@ export default function(context) {
     });
 
     const tocStickyNav = document.querySelector(".publication__toc");
+    const mobileStickyNav = document.querySelector(".mobile__toc");
```

Comment:
For some reason on https://pr-1254-wriflagship.pantheonsite.io/cities the mobile sticky menu hops up past the "Happening Now" section pretty abruptly.

I guess that happens on https://pr-1254-wriflagship.pantheonsite.io/aqueduct as well.

Reply:
That's because the menu goes from being inline to fixed, so I have to add the extra margin height to the section it gets pulled out of. Did it for mobile but forgot that the desktop menus are 42px taller. (Note: I didn't change the transition point for logged-in vs anon, so it goes up under the admin menu before it triggers. Let me know if you want that added).